### ISSUE TITLE
./run.sh make syntax missing privileges

### ISSUE THREAD

**Issue by matentzn** [INCATools/ontology-development-kit#196]
Since the move to the new Makefile, the `./run.sh make` command syntax does not work anymore, I think because run.sh does not have +x executable privileges.

**Comment by matentzn** [comment]
yeah it seems like user permissions are not copied. @cmungall if this is important to you, we could either do answer 1 or 2 from here:
https://stackoverflow.com/questions/19787348/copy-file-keep-permissions-and-owner

If you tell me which, I can do it as well.

**Comment by cmungall** [comment]
`shutil.copy2`
